instant-segment/src/lib.rs

```diff
@@ -103,7 +103,7 @@ impl Segmenter {
             Some((uni, bi_scores)) => (uni, bi_scores),
             // Penalize words not found in the unigrams according
             // to their length, a crucial heuristic.
-            None => return 1.0 - self.uni_total_log10 - word.len() as f64
+            None => return 1.0 - self.uni_total_log10 - word.len() as f64,
```

Comment:
Commit history nit: please squash this into the originating commit using `git rebase -i` (or https://github.com/tummychow/git-absorb).